components/Icon.tsx

```diff
@@ -0,0 +1,50 @@
+type Props = {
+  type: string;
+};
+
+export default function Icon({ type }: Props) {
+  function setType() {
+    if (type === "home") {
+      return (
+        <>
+          <svg
+            className="h-7 w-7"
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            stroke="currentColor"
+            strokeWidth={1.5}
+            viewBox="0 0 24 24"
+            type="home"
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              d="m2.25 12 8.954-8.955a1.126 1.126 0 0 1 1.591 0L21.75 12M4.5 9.75v10.125c0 .621.504 1.125 1.125 1.125H9.75v-4.875c0-.621.504-1.125 1.125-1.125h2.25c.621 0 1.125.504 1.125 1.125V21h4.125c.621 0 1.125-.504 1.125-1.125V9.75M8.25 21h8.25"
+            />
+          </svg>
+        </>
+      );
+    } else if (type === "search") {
+      return (
+        <>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            fill="none"
+            viewBox="0 0 24 24"
+            strokeWidth={1.5}
+            stroke="currentColor"
+            className="w-5 h-5 absolute bottom-5 left-5 text-black"
```

Comment:
definitely don't want these styles inside your generic `Icon` component. What happens if you need to use the search icon somewhere else, but don't want it positioned absolutely like this?

I'd add a `className` prop to the Icon component and append it to the default classes you have for your icons (should only be width and height by default). 

[I'd use a library like clsx to do this.](https://www.npmjs.com/package/clsx) Will be super helpful once you get more into dynamic styles too.

Reply:
also, make sure you check out the [tailwind support section](https://github.com/lukeed/clsx#tailwind-support) when you add it

Reply:
Implemented this but cannot seem to get my icons to react (change position) when I send them the positionProps object. I have no errors.